MetadataDevelopment/ImplementationChecks/implementationCheckEBML_micro.xsl

```diff
@@ -418,6 +419,12 @@
                         <xsl:with-param name="x" select="mmt:MicroMediaTrace/mmt:b[@n='Segment']/mmt:b[@n='Cluster']/mmt:b[@n='SimpleBlock']/mmt:b[@parser='FFV1']/mmt:d[@n='colorspace_type']"/>
                         <xsl:with-param name="list">0 1</xsl:with-param>
                       </xsl:call-template>
+                      <xsl:call-template name="data_is_in_list">
+                        <xsl:with-param name="icid">FFV1-VALID-VERSION</xsl:with-param>
+                        <xsl:with-param name="version">1</xsl:with-param>
+                        <xsl:with-param name="x" select="$ffv1version"/>
+                        <xsl:with-param name="list">0 1 3.4</xsl:with-param>
```

Comment:
we should accept 0, 1, 3.x with x >=4, as e.g. 3.5 is valid and expected to be compatible with our parser (backward compatibility).